### PR TITLE
Fix loading of private key from file

### PR DESCRIPTION
**Pull request by aknuds1**
Fix loading of private key from file, i.e. when `--id` flag is used. My patch ensures that the private key file is loaded into a buffer that's passed on to peer-id, as required.

### CODE REVIEW

**Comment by jacobheun** [comment]
Can you add a test to verify the key is getting read/set properly?

**Comment by aknuds1** [comment]
@jacobheun Done!

File: src/libp2p.js [hunk 1]
```diff
@@ -397,8 +392,6 @@ const createLibp2p = async ({
   hostAddrs,
   announceAddrs,
   dht,
-  dhtClient,
-  connMgr,
```
Comment (jacobheun) [inline src/libp2p.js]: Were these lines removed for a reason? Other than that, this looks good!

Comment (aknuds1) [reply src/libp2p.js]: @jacobheun I removed them since they weren't used. Thought it was good code hygiene. If you want, I can put them back :)

Comment (jacobheun) [reply src/libp2p.js]: Nope that's fine, just wanted to make sure :)

**Comment by jacobheun** [comment]
0.2.2 is released with all the latest changes.

**Pull request merged**